tidy3d/components/medium.py

```diff
@@ -756,8 +993,94 @@ def pole_residue(self):
         )
 
 
+IsotropicMediumType = Union[Medium, PoleResidue, Sellmeier, Lorentz, Debye, Drude]
+
+
+class AnisotropicMedium(AbstractMedium):
```

Comment:
Not sure why this change is here, maybe it needs to be rebased against pre/1.9.0?